react-frontend/src/components/StyleComponents/htmlTags.js

```diff
@@ -146,6 +146,10 @@ export const NavBarHeaderLink = styled.a.attrs({
   font-style: bold;
   line-height: 1;
   padding: 5px;
+  margin-left: 3rem;
+  @media screen and (max-width: 800px) {
```

Comment:
I believe that this is also messing with the alignment of the logo and nav bar on small screens